genesyscloud/resource_genesyscloud_telephony_didpool_test.go

```diff
@@ -0,0 +1,115 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v48/platformclientv2"
+)
+
+func TestAccResourceDidPoolBasic(t *testing.T) {
+	var (
```

Comment:
The var here is extraneous and is usually not done in most go code I see.  I would do:

didPoolResource1         := "test-didpool1"
didPoolStartPhoneNumber1 := "+13175550000"
didPoolEndPhoneNumber1   := "+13175550005"
didPoolDescription1      := "Test description"
didPoolComments1         := "Test comments"
didPoolProvider1         := "PURE_CLOUD"

Reply:
Hi John the var was for aligning the assignment operator but I changed it for now to remove the var grouping

Reply:
No worries.  What you did wasnt wrong, it just isnt idiomatic go.